gramps/gen/utils/alive.py

```diff
@@ -86,6 +86,7 @@ def __init__(self,
         self.MAX_SIB_AGE_DIFF = max_sib_age_diff
         self.MAX_AGE_PROB_ALIVE = max_age_prob_alive
         self.AVG_GENERATION_GAP = avg_generation_gap
+        self.plist = []
```

Comment:
If you made this a set() rather than a list(), then when you check for a containing person, it can happen in constant time, rather than a sequential search.